timestampvm/service.go

```diff
@@ -89,7 +90,7 @@ func (s *Service) GetBlock(_ *http.Request, args *GetBlockArgs, reply *GetBlockR
 		return errNoSuchBlock
 	}
 
-	block, ok := blockInterface.(*Block)
+	block, ok := blockInterface.(*TimeBlock)
```

Comment:
I think we should be able to avoid all type assertions (other than with caches) because we only have one block type. I think we should just call `getBlock` rather than `GetBlock` (and maybe we can simplify the return type of `getBlock`).